node/core/pvf/src/execute/queue.rs

```diff
@@ -155,6 +176,47 @@ impl Queue {
 			purge_dead(&self.metrics, &mut self.workers).await;
 		}
 	}
+
+	fn next_job_index(&self, idle_worker: Option<Worker>) -> Option<usize> {
```

Comment:
Please document the functions in this file, at least the new ones. I know there weren't many docs here to begin with, but having them would be really nice.